src/System.Reflection.Metadata/tests/Metadata/HandleTests.cs

```diff
@@ -39,7 +39,6 @@ public void HandleConversionGivesCorrectKind()
             assert(default(FieldDefinitionHandle), HandleKind.FieldDefinition);
             assert(default(EventDefinitionHandle), HandleKind.EventDefinition);
             assert(default(PropertyDefinitionHandle), HandleKind.PropertyDefinition);
-            assert(default(StandaloneSignatureHandle), HandleKind.StandaloneSignature);
```

Comment:
Nice catch on the dupes. For a minute I was baffled by diff w/o expanding context.